addons/account/models/account_reconcile_model.py

```diff
@@ -92,12 +94,15 @@ class AccountReconcileModel(models.Model):
     second_label = fields.Char(string='Second Journal Item Label')
     second_amount_type = fields.Selection([
         ('fixed', 'Fixed'),
-        ('percentage', 'Percentage of amount')
+        ('percentage', 'Percentage of amount'),
```

Comment:
rename: Percentage of balance and remove the remaining options. In order to use the remaining user will have to create a second line with percentage 100% of balance.